lib/runtime-core/src/vm.rs

```diff
@@ -92,6 +92,8 @@ pub struct InternalCtx {
 
     pub memory_base: *mut u8,
     pub memory_bound: usize,
+
+    pub internals: *mut [u64; INTERNALS_SIZE], // TODO: Make this dynamic?
```

Comment:
Should we consider embedding array?

Reply:
Internals are intended to be statically allocated to middleware modules. I think a fixed-size array would be suitable for this case.